starboard/nplb/compiler_compliance/cpp_compiler_version_check.cc

```diff
@@ -14,7 +14,6 @@
 
 #include <iostream>
 
-#ifdef __clang__
```

Comment:
Won't this fail on our old gcc-6-3 and clang-3-9 builds?

Reply:
no, cobalt on those platforms is built modularly(with cobalt toolchain)